### PR TITLE
Added sanity test to compare any rule changes made by the current model

### PR DESCRIPTION
**Pull request by catalinzpop**
Added a sanity test to check changes made by the current model. Easiest way to test this is: 
-Change one of the rules by either removing some tag assignments or removing the rule completely in stressmodel.js
-Run './bin/generate-base-test-file' (if this fails with a permission error run 'chmod +x bin/generate-base-test-file')
-Undo the changes you've made to the stressmodel.js
-Run './bin/test'
-Observe the rule counts and the differences to the rules
-Hit 'y' on the prompt asking to update the base test file in order to update it

### CODE REVIEW

**Comment by catalinzpop** [comment]
[Create sanity tests for the stress model output](https://trello.com/c/LAUYdNfq/543-create-sanity-tests-for-the-stress-model-output)

**Review by JacksonZeitspace — APPROVED** [review]

**Review by brentlintner — COMMENTED** [review]
I noticed `test/baseRules.json` is a single line. So it will effectively be changed every time.

This means the repository history and size will grow indefinitely by 2.4MB every time we do.

Say we update it 4 times every week, then we will hit a non ideal [1GB](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#repository-size-limits) in ~2 years? 🤔 😳

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "ajv": "^8.11.0",
     "commander": "^9.2.0",
     "dotenv": "^16.0.0",
+    "json-diff": "https://github.com/catalinzpop/json-diff",
```
Comment (brentlintner) [inline package.json:28]: For consistency we should probably fork this under Zeitspace instead.

**Review by brentlintner — CHANGES_REQUESTED** [review]
This is awesome. I have a few adjacent concerns though. Let me know what you think? 🤷 😄 

Side note: about the nitpicks during demo (lol), while it would be cool to see correct numerical sorting, and even colors + more pretty printing... perhaps we should make a Tech Debt item for later? 🤔

File: ltsanalyzer.js [hunk 2]
```diff
@@ -160,7 +179,87 @@ export default class ltsanalyzer {
...
+        } else {
+          counts[key] = [0, Object.keys(orderedRuleData[key]).length];
+        }
+      }
+
+      for (let lts of Object.keys(this.LTScolour).sort()) {
+        for (let key in orderedRuleData) {
+          if (lts == ruleLTS[key]) {
+            const delta = counts[key][1] - counts[key][0];
+            console.log(`${key.length == 2 ? key + ' ' : key} ${this.LTScolour[ruleLTS[key]]('(LTS ' + (ruleLTS[key].toString().length > 1 ? ruleLTS[key] : ' ' + ruleLTS[key]) + ')')} = ${counts[key][1].toString() + this.formattingSpaces[6 - counts[key][1].toString().length]} ${delta != 0 ? delta > 0 ? chalk.green('(+' + delta + ')') : chalk.red('(' + delta + ')') : ''}`);
```
Comment (catalinzpop) [inline ltsanalyzer.js:239]: Sorry for this

File: stressmodel.json [hunk 3]
```diff
@@ -239,11 +239,6 @@
             "tag": "highway",
             "values": ["service"],
             "present": true
-          },
-          {
-            "tag": "service",
-            "values": ["alley"],
-            "present": true
```
Comment (brentlintner) [inline stressmodel.json:246]: Should put this back?

File: package.json [hunk 4]
```diff
@@ -23,8 +23,10 @@
   "homepage": "https://github.com/BikeOttawa/stressmodel#readme",
   "dependencies": {
     "ajv": "^8.11.0",
+    "chalk": "^5.1.2",
     "commander": "^9.2.0",
     "dotenv": "^16.0.0",
+    "json-diff": "https://github.com/zeitspace/json-diff",
```
Comment (brentlintner) [inline package.json:29]: See my double fork comment in slack! 😅

File: ltsanalyzer.js (hunk 2 above)
Comment (brentlintner) [reply ltsanalyzer.js:239]: noooo

File: stressmodel.json (hunk 3 above)
Comment (catalinzpop) [reply stressmodel.json:246]: Yes I did sorry bout that

**Review by brentlintner — APPROVED** [review]
LGTM. 🦅

**Pull request merged**